op-e2e/sequencer_failover_test.go

```diff
@@ -136,3 +136,31 @@ func TestSequencerFailover_ConductorRPC(t *testing.T) {
 	require.Equal(t, 2, len(membership), "Expected 2 members in cluster after removal")
 	require.NotContains(t, membership, fid, "Expected follower to be removed from cluster")
 }
+
+// [Category: Sequencer Failover]
+// Test that the sequencer can successfully failover to a new sequencer once the active sequencer goes down.
+func TestSequencerFailover_ActiveSequencerDown(t *testing.T) {
+	sys, conductors := setupSequencerFailoverTest(t)
+	defer sys.Close()
+
+	ctx := context.Background()
+	leaderId, leader := findLeader(t, conductors)
+	err := sys.RollupNodes[leaderId].Stop(ctx) // Stop the current leader sequencer
+	require.NoError(t, err)
+
+	// The leadership change should occur with no errors
+	newID := waitForLeadershipChange(t, leader, leaderId, conductors, sys)
+	require.NotEqual(t, leaderId, newID, "Expected leader to change")
+
+	// Confirm the new leader is different from the old leader
+	newLeaderId, _ := findLeader(t, conductors)
+	require.NotEqual(t, leaderId, newLeaderId, "Expected leader to change")
+
+	// Check that the sequencer is healthy
+	require.True(t, healthy(t, ctx, conductors[newLeaderId]))
```

Comment:
Is the sequencer guaranteed to be immediately healthy or do we need a waiter/eventually here?

Reply:
@ajsutton yes it is guaranteed to be healthy if selected as the leader! https://github.com/ethereum-optimism/optimism/blob/develop/op-conductor/conductor/service.go#L550-L596

Reply:
yes @ajsutton in this case all but one were healthy and we waited for the leadership change from the 1 unhealthy one to one of the others, so the new leader here should always be healthy. 

Can also see here: https://github.com/ethereum-optimism/optimism/blob/develop/op-conductor/conductor/service.go#L550-L596